src/cluster.ml

```diff
@@ -68,6 +69,7 @@ type service = [
   | `Ocamlorg_images of string               (* Base Image builder @ images.ci.ocaml.org *)
   | `OCamlorg_v3b of string                  (* OCaml website @ v3b.ocaml.org aka www.ocaml.org *)
   | `OCamlorg_v3c of string                  (* Staging OCaml website @ staging.ocaml.org *)
+  | `Dune_distribution_docker of string      (* Dune binary distribution website *)
```

Comment:
```suggestion
  | `Dune_binary_distribution      (* Dune binary distribution website *)
```
We can stick to for now. If the infrastructure gets more complex, we can change the naming at this point/.

Reply:
Yes, I've updated this.  It still needs to be `of string`, though.